temporal-sdk/src/main/java/io/temporal/client/WorkflowOptions.java

```diff
@@ -444,6 +449,17 @@ public Builder setCompletionCallbacks(List<Callback> completionCallbacks) {
       return this;
     }
 
+    /**
+     * Links to be associated with the workflow.
+     *
+     * <p>WARNING: Not intended for User Code.
+     */
+    @Experimental
+    public Builder setLinks(List<Link> links) {
```

Comment:
Works for me as long as it's experimental. @cretz do you have a concern with adding this public API?

Reply:
Yeah ideally I wouldn't expose it, but there is not good way to hide this given the package layout of the Java SDK